typings/objection/index.d.ts

```diff
@@ -562,10 +563,6 @@ declare namespace Objection {
     <RelatedModel extends Model>(
       ids: MaybeCompositeId | Partial<RelatedModel> | Partial<RelatedModel>[]
     ): NumberQueryBuilder<QB>;
-
-    <RelatedModel extends Model>(
```

Comment:
This was a total duplicate of the above types. Unsure if copy-pasta mistake, or intended to be tweaked.

Reply:
Delicious copy-pasta. Thanks